controllers/ocsinitialization/ocsoperator_config.go

```diff
@@ -24,8 +24,9 @@ func (r *OCSInitializationReconciler) ensureOCSOperatorConfig(initialData *ocsv1
 			Namespace: initialData.Namespace,
 		},
 		Data: map[string]string{
-			"CSI_CLUSTER_NAME":         r.getClusterID(),
-			"CSI_ENABLE_READ_AFFINITY": "true",
+			"CSI_CLUSTER_NAME":                r.getClusterID(),
+			"CSI_ENABLE_READ_AFFINITY":        "true",
+			"CSI_CEPHFS_KERNEL_MOUNT_OPTIONS": "ms_mode=prefer-crc",
```

Comment:
Is this required to be set? Because `prefer-crc` is getting set in setCephFSKernelMountOption also.

Reply:
Going forward ocsinitializer will just create the configmap with the keys so that rook operator doesn't through errors. (as rook operator expects these value so if the configmap or the key doesn't exist it will stay in container create config error). So we need to create the configmap with some default value when the operator is installed & we can later update them with correct values in storagecluster controller.